Chapter02/recipe-0011/README.md

```diff
@@ -0,0 +1,100 @@
+# Discovering the operating system
+
+CMake is a set of cross-platform tools. Nevertheless it can be very useful to
+know on which operating system (OS) the configure or build step is executed:
+either to tweak CMake code for a particular OS, to enable conditional
+compilation depending on the OS or to use compiler-specific extensions if
+available. In this recipe we will demonstrate how to use CMake to detect the
+OS.
+
+
+## Ingredients
+
+- A C++ compiler
+- CMake (any version higher than 3.0)
+
+
+## How to do it
```

Comment:
In "How to do it" we need to have a step-by-step comment of the `CMakeLists.txt`. I've used numbered lists, see start-up guide.

Reply:
Roger. Saw that but somehow did it differently but will realign.

Reply:
To me the numbered lists look actually strange: N) describing a step and N+1) showing the step making it look as if these were two steps. If numbered, then the steps should be well defined atomic steps.

Reply:
The N+1 is because I can't get the code blocks to intersperse correctly in the list. The editor will take care of that, I hope.